code/modules/surgery/surgery.dm

```diff
@@ -76,27 +76,29 @@ proc/spread_germs_to_organ(datum/limb/E, mob/living/carbon/human/user)
 	//Gloves
 	if(user.gloves)
 		if(user.germ_level && istype(user.gloves, /obj/item/clothing/gloves/latex))
-			E.germ_level -= user.gloves.germ_level * 0.5
+			E.germ_level += user.gloves.germ_level * 0.5
```

Comment:
The behavior is identical to the `else`, making it moot. What's the logic here? Should latex gloves pass a smaller fraction of the germs than other gloves? Or pass no germs at all?

Reply:
The way the infection calculations work is that you have to reduce it like this, but yeah fair point for the numbers, I've tweaked them.